libs/core/src/main/java/org/opensearch/core/xcontent/MediaType.java

```diff
@@ -42,7 +43,10 @@
  * Abstracts a <a href="http://en.wikipedia.org/wiki/Internet_media_type">Media Type</a> and a format parameter.
  * Media types are used as values on Content-Type and Accept headers
  * format is an URL parameter, specifies response media type.
+ *
+ * @opensearch.api
  */
+@PublicApi(since = "1.0.0")
```

Comment:
```suggestion
@PublicApi(since = "2.1.0")
```

Reply:
I didn't introduce the more general `MediaType` interface [until 2.1](https://github.com/opensearch-project/OpenSearch/pull/3077)